packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -60,56 +53,66 @@ const saveSnapshotsForFile = (
   snapshots: Array<InlineSnapshot>,
   sourceFilePath: Config.Path,
   prettier: any,
-  babelTraverse: Function,
+  babelTraverse: BabelTraverse,
 ) => {
   const sourceFile = fs.readFileSync(sourceFilePath, 'utf8');
 
-  // Resolve project configuration.
-  // For older versions of Prettier, do not load configuration.
-  const config = prettier.resolveConfig
-    ? prettier.resolveConfig.sync(sourceFilePath, {
-        editorconfig: true,
-      })
-    : null;
-
-  // Detect the parser for the test file.
-  // For older versions of Prettier, fallback to a simple parser detection.
-  const inferredParser = prettier.getFileInfo
-    ? prettier.getFileInfo.sync(sourceFilePath).inferredParser
-    : (config && config.parser) || simpleDetectParser(sourceFilePath);
+  // TypeScript projects may not have a babel config; make sure they can be parsed anyway.
+  const plugins = [];
+  if (/\.tsx?$/.test(sourceFilePath)) {
+    plugins.push([
+      require.resolve('@babel/plugin-syntax-typescript'),
+      {isTSX: /\.tsx$/.test(sourceFilePath)},
```

Comment:
```suggestion
      {isTSX: sourceFilePath.endsWith('x')},
```

no? it's inside an if which looks identical expect for optional `x` at the end

Reply:
Yeah that's a simpler variant I supposed :+1: